### PR TITLE
Calculator app milestone three

### PR DESCRIPTION
**Pull request by che30**
In this milestone the following was carried out :
- I added a logic folder in src folder
- I added a calculate.js file with calculate method
- I added an operate.js file with a operate method

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    case '+':
+      total = bigNumberOne + bigNumberTwo;
+      result = { total, next: total, operator };
+      break;
+    case 'X':
+      total = bigNumberOne * bigNumberTwo;
+      result = { total, next: total, operator };
+      break;
+    case '%':
+      return { total: bigNumberOne, next: bigNumberTwo / 100, operator };
```
Comment (bellom) [inline src/logic/operate.js]: Job well done but you don't have to do the calculating yourself, that where the use of `big` comes in -> [big js library](https://github.com/favio41/big-js#readme)
example: case division will be -> `total.minus(next)`

File: src/logic/calculate.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+const Calculate = ({ total, next, operator }, buttonName) => {
+  switch (buttonName) {
+    case '+/-':
+      return { total: total * -1, next: next * -1, operator };
+    case '%':
+      return operate(total, next, buttonName);
+    case 'AC':
+      return { total: 0, next: 0, operator };
+    default:
+      return operate(total, next, buttonName);
```
Comment (bellom) [inline src/logic/calculate.js]: ### Heads-Up:
**we need to add more cases;** 
example;
 - if the `buttonName`  = `.` we should return `0.`
 - case `/`, `x`, and `=`

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @che30  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations @che30  :tada: :clinking_glasses: 

**Thank you for making all the required changes. This milestone has passed all the mandatory requirements :clap: 
It is therefore approved :heavy_check_mark:**


Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**